### ISSUE TITLE
Error use alpha matting as library

### ISSUE THREAD

**Issue by ridwanchaniago84** [danielgatis/rembg#251]
Hello sir, when i use this library with alpha matting i get error
```
from rembg import remove

input_path = 'input.png'
output_path = 'output.png'

with open(input_path, 'rb') as i:
    with open(output_path, 'wb') as o:
        input = i.read()
        output = remove(input, True)
        o.write(output)
```
![3](https://user-images.githubusercontent.com/69184610/173210500-b3d63d52-74ee-4bfe-bb78-419e332c1fdb.png)

**Comment by ridwanchaniago84** [comment]
Fixed when i downgrade to 2.0.13 version

**Comment by juergengunz** [comment]
+1 same issue
